doc/languages-frameworks/beam.section.md

```diff
@@ -52,15 +54,151 @@ Erlang.mk functions similarly to Rebar3, except we use `buildErlangMk` instead o
 
 #### Mix Packages {#mix-packages}
 
-Mix functions similarly to Rebar3, except we use `buildMix` instead of `buildRebar3`.
+`mixRelease` is used to make a release in the mix sense. Dependencies will need to be fetched with `fetchMixDeps` and passed to it.
+
+#### mixRelease - Elixir Phoenix example
+
+Here is how your `default.nix` file would look.
+
+```nix
+with import <nixpkgs> { };
+
+let
+  packages = beam.packagesWith beam.interpreters.erlang;
+  src = builtins.fetchgit {
+    url = "ssh://git@github.com/your_id/your_repo";
+    rev = "replace_with_your_commit";
```

Comment:
That should probably be fetchFromGitHub or at least use the anonymous https url.

Reply:
I used fetchgit because I suspect the majority of cases would be a private repo. My understanding is that the only way to fetch a private repo is with fetchgit and the ssh url.
Let me know if I'm wrong here.

Reply:
To be honest I am not sure but this is fine for now.